x/ibc/04-channel/client/utils/utils.go

```diff
@@ -218,6 +228,11 @@ func queryNextSequenceRecvABCI(clientCtx client.Context, portID, channelID strin
 		return nil, err
 	}
 
+	// check if next sequence receive exists
+	if len(value) == 0 {
+		return nil, sdkerrors.Wrapf(types.ErrChannelNotFound, "portID (%s), channelID (%s)", portID, channelID)
```

Comment:
I'm pretty sure it only doesn't exist when the channel doesn't exist, hence the error